recipe/0001-remove-pybind-fetch.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index d97ca9a..4bf4eb4 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -32,15 +32,18 @@ install(FILES ${_dpctl_capi_headers}
+ # Define CMAKE_INSTALL_xxx: LIBDIR, INCLUDEDIR
+ include(GNUInstallDirs)
+ 
+-# Fetch pybind11
+-include(FetchContent)
++# Anaconda: Use pybind11 from default instead
++find_package(pybind11 REQUIRED)
+ 
+-FetchContent_Declare(
+-    pybind11
+-    URL https://github.com/pybind/pybind11/archive/refs/tags/v2.10.2.tar.gz
+-    URL_HASH SHA256=93bd1e625e43e03028a3ea7389bba5d3f9f2596abc074b068e70f4ef9b1314ae
+-)
+-FetchContent_MakeAvailable(pybind11)
++# Fetch pybind11
++#include(FetchContent)
++
++#FetchContent_Declare(
++#    pybind11
++#    URL https://github.com/pybind/pybind11/archive/refs/tags/v2.10.2.tar.gz
```

Comment:
in the host there is pybind 2.10.4 but here is 2.10.2 ?

Reply:
We had to release pybind11 2.10, so instead of releasing an older version, we went with the latest. Any version pf Pybind11 2.10 will do it.